scripts/rollup/validate/index.js

```diff
@@ -36,7 +37,43 @@ function run(format, filePatterns) {
   }
 }
 
-run('fb', [`./build/facebook-www/*.js`]);
-run('rn', [`./build/{react-cs,react-native,react-rt}/*.js`]);
-run('umd', [`./build/packages/*/umd/*.js`]);
-run('cjs', [`./build/packages/*/*.js`, `./build/packages/*/cjs/*.js`]);
+function checkFilesExist(bundle) {
+  const {format, filePatterns} = bundle;
+  filePatterns.map(pattern => {
+    console.log(`Check if files exist in ${pattern}`);
+    const files = glob.sync(pattern);
+    if (files.length === 0) {
+      console.error(
+        chalk.red(
+          `No files found in glob pattern ${pattern} in ${format} bundle.`
+        )
+      );
+      process.exit();
```

Comment:
Note this doesn't actually fail the build. Should be `process.exit(1)` instead.
I fixed in https://github.com/facebook/react/commit/646781b0b4f95d90cf08b7799bbd196e16136f06.

Reply:
Thank you for the review! Well noted 😄